0/setup.md

```diff
@@ -9,12 +9,17 @@ Follow the
 [official installation steps](https://substrate.dev/docs/en/knowledgebase/getting-started/) from the
 Substrate Developer Hub Knowledge Base.
 
+```
+rustup target add wasm32-unknown-unknown --toolchain stable
+rustup component add rust-src --toolchain nightly
+```
+
 ## Installing A Substrate Node
 
 We need to use a Substrate node with the built-in Contracts module. For this workshop we'll use the pre-designed Substrate node client.
 
 ```bash
-cargo install node-cli --git https://github.com/paritytech/substrate.git --tag v2.0.0 --force --locked
+cargo install canvas-node --git https://github.com/paritytech/canvas-node.git --force
```

Comment:
Are we going to make this repo public?

Reply:
I was telling this to @Robbepop If not, we end up with this issue https://github.com/paritytech/canvas-ui/issues/6

Reply:
Yep let us make it public.

Reply:
That's an idea but I think it is better to provide a built-in solution to not require this. We could still provide a tip to those who already went through that other tutorial that they can also optionally use their own chain.

Reply:
https://github.com/paritytech/canvas-node/releases/tag/v0.1.0